#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_no_py26_dependency.dpatch by Torsten Marek <shlomme@debian.org>
##
## DP: Do not require Python 2.6, Python 2.5 still works just fine

@DPATCH@
diff -urNad pycairo-1.8.4~/setup.py pycairo-1.8.4/setup.py
--- pycairo-1.8.4~/setup.py	2009-03-19 09:44:29.000000000 +0100
+++ pycairo-1.8.4/setup.py	2009-03-23 22:56:08.000000000 +0100
@@ -32,8 +32,8 @@
     return [x.lstrip(opt) for x in output.split()]
 
 
-if sys.version_info < (2,6):
-    raise SystemExit('Error: Python >= 2.6 is required')
+#if sys.version_info < (2,6):
+#    raise SystemExit('Error: Python >= 2.6 is required')
 
 pkg_config_version_check ('cairo', cairo_version_required)
 if sys.platform == 'win32':
